tests/installation/ipxe_install.pm

```diff
@@ -182,7 +182,7 @@ sub run {
     select_console 'sol', await_console => 0;
 
     # Print screenshots for ipxe boot process
-    if (get_var('VIRT_AUTOTEST') || get_var('HANA_PERF')) {
+    if (get_var('VIRT_AUTOTEST')) {
```

Comment:
If you just want to exclude your jobs, you can do it, that's fine.

Reply:
Thanks!